error.go

```diff
@@ -90,7 +90,7 @@ func (e *Err) Underlying() error {
 	return e.previous
 }
 
-// The Cause of an error is the most recent error in the error stack that
+// Cause of an error is the most recent error in the error stack that
```

Comment:
I think this would read more naturally as `Cause returns the most recent error in the error stack ...`

Reply:
Good idea.